src/main/kotlin/com/hedvig/accountservice/schedulers/web/ScheduleController.kt

```diff
@@ -128,6 +133,31 @@ class ScheduleController @Autowired constructor(
         return ResponseEntity.noContent().build()
     }
 
+    @PostMapping("/approvals/auto")
+    fun attemptAutoApproveChargeAmounts(
+        @RequestParam approvedBy: String
+    ): ResponseEntity<Void> {
+        val memberIdsWaitingForApproval = stateManager.findAllMemberIds(ChargeStatus.SUBSCRIPTION_SCHEDULED_AND_WAITING_FOR_APPROVAL)
```

Comment:
is it better to create a specific method without passing the enum? 

Something like this

```suggestion
        val memberIdsWaitingForApproval = stateManager.findAllMemberIdsWaitingForApproval()
```